cmd/postpolicyform.go

```diff
@@ -240,7 +240,7 @@ func checkPostPolicy(formValues http.Header, postPolicyForm PostPolicyForm) erro
 	}
 
 	// Flag to indicate if all policies conditions are satisfied
-	condPassed := true
+	var condPassed bool
```

Comment:
this changes the starting value to `false`. The previous version of the code possibly relies on `condPassed` being `true`.

Reply:
No it doesn't @krisis